src/main/java/seedu/address/model/person/Person.java

```diff
@@ -127,6 +127,31 @@ public Address getAddress() {
         return address;
     }
 
```

Comment:
I moved the following methods up in the Person class to group them with the other getter methods.